src/main/resources/migrations.xml

```diff
@@ -2234,4 +2234,40 @@
 			onDelete="NO ACTION" onUpdate="NO ACTION" />
 	</changeSet>
 
+	<changeSet id="renameApprovalActions-to-ReportActions-sqlite-combined-tables" author="maradragan" dbms="mssql,postgresql">
+		<comment>Rename approvalActions table to reportActions.</comment>
+		<!-- reportActions -->
+		<renameTable
+			newTableName="reportActions"
+			oldTableName="approvalActions" />
+		<rollback>
+			<renameTable
+				newTableName="approvalActions"
+				oldTableName="reportActions" />
+		</rollback>
+	</changeSet>
+
+	<changeSet id="migrate_submit_and_publish_action" author="maradragan" dbms="mssql">
```

Comment:
Shouldn't there also be a migration to change the state of all previously RELEASED reports (`state = 2`) to APPROVED (`state = 6`)?